src/pycountry/db.py

```diff
@@ -62,31 +64,28 @@ class Subdivision(Data):
     pass
 
 
-def lazy_load(f):
+def lazy_load(f: F) -> F:
```

Comment:
Let mypy or other type checkers know that they should consider whatever to be returned to match whatever was passed in. I can also call this `T` since it's doesn't have a lower bound.